### ISSUE TITLE
Types missing in "exports"

### ISSUE THREAD

**Issue by chrisvander** [Ivordir/YALPS#4]
When importing YALPS, I get the following issue:

There are types at '/.../node_modules/yalps/lib/types/index.d.ts', but this result could not be resolved when respecting package.json "exports". The 'yalps' library may need to update its package.json or typings.

**Comment by chrisvander** [comment]
Quick update on this: I patched `package.json` so that "types" are in the "exports" object. I also nested it under a ".", but unclear if that's required.

```ts
{
  "exports": {
    ".": {
      "types": "./lib/types/index.d.ts",
      "import": "./lib/module/index.js",
      "require": "./lib/main/index.js",
      "default": "./lib/module/index.js"
    }
  }
}
```

**Comment by Ivordir** [comment]
Thanks for bringing this to my attention! It looks like typescript updated/added support for `exports` a while back and I missed it. If you can, could you confirm if the patch below also works? (I think the `"."` is not strictly necessary.)
```patch
"exports": {
+   "types": "./lib/types/index.d.ts",
    "import": "./lib/module/index.js",
    "require": "./lib/main/index.js",
    "default": "./lib/module/index.js"
  },
```

**Comment by Ivordir** [comment]
Well, I published the change with version 0.5.6. If anyone still has issues, just let me know.
